### PR TITLE
test: added tests

### PR DESCRIPTION
**Pull request by cachho**
some of these are smart, some are more simple. But better than nothing I guess.

### CODE REVIEW

File: tests/chunkers/test_text.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return [
+            {
+                "content": src,
+                "meta_data": {"url": "none"},
+            }
+        ]
+
+
+if __name__ == "__main__":
+    unittest.main()
```
Comment (deshraj) [inline tests/chunkers/test_text.py:46]: You don't need this I think.

File: tests/chunkers/test_text.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# ruff: noqa: E501
```
Comment (deshraj) [inline tests/chunkers/test_text.py:1]: Any specific reason to add this? :D

File: tests/embedchain/test_add.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# ruff: noqa: E501
```
Comment (deshraj) [inline tests/embedchain/test_add.py:1]: Let's remove this from all the test files. We only want to specific specific lines and not all of the file.

File: tests/chunkers/test_text.py (hunk 2 above)
Comment (cachho) [reply tests/chunkers/test_text.py:1]: otherwise it complains about all the doc strings.

File: tests/embedchain/test_add.py (hunk 3 above)
Comment (cachho) [reply tests/embedchain/test_add.py:1]: that's going to be a lot of lines and decrease readability.

**Comment by taranjeet** [comment]
@deshraj waiting for your review, since you are driving adding tests feature.

**Review by deshraj — APPROVED** [review]

**Pull request merged**